unit_tests/util/cell_customization.cpp

```diff
@@ -0,0 +1,277 @@
+#include <boost/numeric/conversion/cast.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "customizer/cell_customizer.hpp"
+#include "partition/cell_storage.hpp"
+#include "partition/multi_level_partition.hpp"
+#include "util/static_graph.hpp"
+
+#define REQUIRE_SIZE_RANGE(range, ref) BOOST_REQUIRE_EQUAL(range.size(), ref)
+#define CHECK_EQUAL_RANGE(range, ...)                                                              \
```

Comment:
This is cool, could we move this into a common header? We could reuse this for the `CellStorage` tests.